packages/openauth/src/client.ts

```diff
@@ -38,22 +38,22 @@
  *
  * @packageDocumentation
  */
+import type { v1 } from "@standard-schema/spec"
```

Comment:
These got reordered because of `Organize Imports`. Let me know if you want me to revert this part.